website/publish/multi_website.rst

```diff
@@ -37,25 +37,25 @@ Then, select a theme. This new website might have an entirely different
 purpose or audience than the first one. So feel free to go for a
 different theme!
 
-Once the theme is selected, you can start to build the homepage of your
+Once the theme is selected, you can start to build the homepage for your
 website. Follow the purple drops; they will help you in the first steps.
 
-.. image:: media/multi_website08.png
```

Comment:
Is this image file still referenced somewhere else? If not, it should be removed.

Reply:
I no longer see this on line 43. I am assuming it has been removed. it was replaced by the image that I need to retitle and compress.

Reply:
Indeed the *reference* to the image was removed, but not the image file itself :p

PS: Usually we either answer a conversation or resolve it, but rarely both. This allows reading answers without expanding all closed conversations ;)